### PR TITLE
Rework README.md

### PR DESCRIPTION
**Pull request by BenediktSeidl**
Split README.md into DEVELOPMENT.md, CHANGELOG.md and the official
docs.

### CODE REVIEW

**Review by PhilippSiegmantel — APPROVED** [review]
Looks great to me!

File: README.md [hunk 1]
```diff
@@ -2,177 +2,27 @@
...
-This data-source plugin is a complete rewrite from the previous connector. It has
-undergone a new architectural design and is not backwards compatible to the
-previous version. We nevertheless provide an upgrade procedure to assist you on
-the transition.
+This is a [Grafana][1] [data source][2] plugin for working with checkmk metrics
+in Grafana.
 
-This project has entered the Beta testing phase.
+The plugin requires checkmk >= 2.1.0 and Grafana >= 8.0  
+(It is possible query data from checkmk > 2.0.0p20 but it's not recommended.)
```
Comment (BenediktSeidl) [inline README.md:9]: what exactly?

File: README.md [hunk 2]
```diff
@@ -2,177 +2,27 @@
 
 ![CI](https://github.com/tribe29/grafana-checkmk-datasource/actions/workflows/node.js.yml/badge.svg)
 
-This data-source plugin is a complete rewrite from the previous connector. It has
-undergone a new architectural design and is not backwards compatible to the
-previous version. We nevertheless provide an upgrade procedure to assist you on
-the transition.
+This is a [Grafana][1] [data source][2] plugin for working with checkmk metrics
```
Comment (BenediktSeidl) [inline README.md:5]: .. for visualizing checkmk metrics in Grafana.

File: README.md [hunk 3]
```diff
@@ -2,177 +2,27 @@
...
+in Grafana.
 
-This project has entered the Beta testing phase.
+The plugin requires checkmk >= 2.1.0 and Grafana >= 8.0  
+(It is possible query data from checkmk > 2.0.0p20 but it's not recommended.)
 
-## Requirements
+Both Checkmk Enterprise Edition and Checkmk Raw Edition are supported by this
```
Comment (BenediktSeidl) [inline README.md:11]: All Checkmk Editions are supported

File: README.md [hunk 4]
```diff
@@ -2,177 +2,27 @@
...
+plugin, but with different query interface as the Enterprise Edition has a
+better filtering engine.
```
Comment (BenediktSeidl) [inline README.md:13]: better -> more comprehensive

File: README.md [hunk 5]
```diff
@@ -2,177 +2,27 @@
...
 
-- Checkmk >= 2.0.0p20
-- Grafana >= 8.0
+Please consult chapter ["Integrating Checkmk in Grafana"][3] in the official
```
Comment (BenediktSeidl) [inline README.md:15]: -> Please consult the chapter [Integrating Checkmk in Grafana][3] in the official

**Pull request merged**